install_scripts/templates/kubernetes/migrate.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/sh
+
+set -e
+
+TMP_DIR=/tmp/migrate-k8s
+LOG_LEVEL=info
+PRIVATE_ADDRESS=
+PUBLIC_ADDRESS=
+AIRGAP=0
+# If all state needed from the native app has been saved we can skip some steps
+HAS_NATIVE_STATE=0
+HAS_KUBERNETES=0
+HAS_APP=0
+
+{% include 'common/cli-script.sh' %}
+{% include 'common/common.sh' %}
+{% include 'common/log.sh' %}
+{% include 'common/system.sh' %}
+
+startNativeScheduler() {
+    logStep "Ensuring native scheduler is running"
+
+    systemctl start replicated
```

Comment:
Do we need to handle different service managers like [here](https://github.com/replicatedhq/replicated-installer/blob/76f9d7d3b9feb2395ec33f3d5de2b009eae2cf56/install_scripts/templates/replicated-2.0.sh#L740)

Reply:
For now all our supported Kubernetes OS's use systemd

Reply:
I don't see where this script would bail if it's one of unsupported OS...

Reply:
added OS check to validate step